### PR TITLE
html exercise 1

### PR DESCRIPTION
**Pull request by alejandro-facundo**
First version of the exercise,  i'll appreciate the feedback

### CODE REVIEW

File: Exercise-HTML-CSS-1/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  <title>Excercise 1</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+  div{
+    padding:10px;
+  }
+</style>
+</head>
+<body>
+  <div>
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Try to wrap all your content in an initial `container` class from Bootstrap

File: Exercise-HTML-CSS-1/index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <img src ="http://www.girldevelopitfargo.com/images/GDI.jpg">
+    <h1>About</h1>
+    <p>
+      It can be intimidating for women to learn and ask questions when they are in the <strong>extreme minority</strong>strong.
+      While open and welcoming, today's budding developer community is up to <strong>91% male</strong>. If we can empower more females with the confidence in their technological capabilities <em>we can begin to change this landscape</em>.
+    </p>
+    <h1>Our Locations</h1>
+    <ul>
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Try to use a table here!

File: Exercise-HTML-CSS-1/index.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        <div class="w-100"></div>
+        <div class="col">San Francisco</div>
+        <div class="col">HTML & CSS</div>
+      </div>
+    </div>
+
+    <h1>Sign Up!</h1>
+
+    <form>
+      Name: <input type="text" name="name"><br>
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Try to add more Bootstrap style to forms
https://getbootstrap.com/docs/4.1/components/forms/

File: Exercise-HTML-CSS-1/index.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      <fieldset>
+        <legend>Gender:</legend>
+        <input type="radio" name="Gender" value="male">Male
+        <input type="radio" name="Gender" value="female">Female
+      </fieldset>
+      <br>
+      Experience:<br>
+      <textarea rows="1" cols="15">
+      </textarea><br>
+      <input type="submit">
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Try to add more Bootstrap style to buttons
https://getbootstrap.com/docs/4.1/components/buttons/

**Comment by alejandro-facundo** [comment]
Made some changes using bootstrap as requested

File: index.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    <div>
+      Name: <input type="text" name="fname"><br>
+      Email: <input type="text" name="fmail"><br>
+      Location: <select>
+        <option>San Francisco</option>
+      </select><br>
+      Gender: Female <input type="radio" name="sex"> Male <input type="radio" name="sex"><br>
+      Experience:<br>
+      <textarea></textarea><br>
+      <button>Submit</button>
```
Comment (AlexHenkel) [inline index.html]: Try some Bootstrap style buttons
https://getbootstrap.com/docs/4.1/components/buttons/

File: index.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+      </div>
+      <div class="row justify-content-sm-left">
+        <div class="col-sm-2">San Francisco</div>
+        <div class="col-sm-2">HTML & CSS</div>
+      </div>
+    </div>
+
+    <h1>Sign Up!</h1>
+
+    <div>
```
Comment (AlexHenkel) [inline index.html]: Wrap everything in a `<form>` tag and add some Bootstrap style
https://getbootstrap.com/docs/4.1/components/forms/

File: index.html [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+    <div class="table-responsive">
+      <table class="table">
+        It can be intimidating for women to learn and ask questions when they are in the <strong>extreme minority</strong>strong.
+        While open and welcoming, today's budding developer community is up to <strong>91% male</strong>. If we can empower more females with the confidence in their technological capabilities <em>we can begin to change this landscape</em>.
+        
+      </table>
+    </div>
+    <h1>Our Locations</h1>
+    <ul>
```
Comment (AlexHenkel) [inline index.html]: Table should be used here

File: index.html [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+    div{
+      padding:10px;
+    }
+  </style>
+</head>
+<body>
+  <div>
+    <img src ="http://www.girldevelopitfargo.com/images/GDI.jpg">
```
Comment (AlexHenkel) [inline index.html]: Try some image classes
https://getbootstrap.com/docs/4.1/content/images/

File: index.html [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+  <html>
+  <head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>    
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Excercise 1</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+    div{
```
Comment (AlexHenkel) [inline index.html]: Move all CSS to an external file, by following file structure specified in the exercise

File: index.html [hunk 10]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<div class="container">
```
Comment (AlexHenkel) [inline index.html]: Remove this `<div>`, you should not have any div outside the `<body>` tag

**Comment by alejandro-facundo** [comment]
Changed directory and added style to the logo and button

File: Exercise-HTML-CSS-1/styles [hunk 11]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/styles]: This file should have the .css extension

File: Exercise-HTML-CSS-1/index.html [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+      <div class="row justify-content-sm-left">
+        <div class="col-sm-2">San Francisco</div>
+        <div class="col-sm-2">HTML & CSS</div>
+      </div>
+    </div>
+
+    <h1>Sign Up!</h1>
+
+    <div>
+      Name: <input type="text" name="fname"><br>
```
Comment (AlexHenkel) [inline Exercise-HTML-CSS-1/index.html]: Missing Bootstrap style inputs. If you have any doubts on that, contact me!

**Pull request closed without merging**